bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/Teaser.java

```diff
@@ -28,6 +31,20 @@
  */
 public interface Teaser extends ComponentExporter {
 
+    /**
+     * Name of the property that will store if the teaser has Call-to-Action elements
+     *
+     * @since com.adobe.cq.wcm.core.components.sandbox.models 1.0.0
+     */
+    String PN_WITH_CTA = "withCTA";
+
+    /**
+     * Name of the node where the Call-to-Action elements will be stored
```

Comment:
I'd write it in present form: 
Name of the parent node where the Call-to-Action elements are stored